src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -29,13 +29,13 @@ module sonic-port{
 
 			list PORT_LIST {
 
-				key "port_name";
+				key "name";
```

Comment:
@praveen-li : Can you please review these changes & approve it? @lguohan has reviewed & approved it. Thanks @lguohan

Reply:
Thanks @praveen-li for the approval. Hope we have enough approvals to merge this PR. Can you please merge this? @praveen-li / @lguohan ?